### PR TITLE
[Bugfix] Wrong return value of HDFSFileInterpreter completion.

### PR DESCRIPTION
**Pull request by astroshim**
### What is this PR for?
This PR fixes auto competion of HDFSFileInterpreter.

### What type of PR is it?
Bug Fix

### How should this be tested?
Please refer to test case.

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: file/src/main/java/org/apache/zeppelin/file/HDFSFileInterpreter.java [hunk 1]
```diff
@@ -311,7 +311,7 @@ public boolean isDirectory(String path) {
                 String beforeLastPeriod = unfinished.substring(0, unfinished.lastIndexOf('.') + 1);
                 //beforeLastPeriod should be the start of fs.pathSuffix, so take the end of it.
                 String suggestedFinish = fs.pathSuffix.substring(beforeLastPeriod.length());
-                suggestions.add(suggestedFinish);
+                suggestions.add(new InterpreterCompletion(suggestedFinish,suggestedFinish));
```
Comment (cloverhearts) [inline file/src/main/java/org/apache/zeppelin/file/HDFSFileInterpreter.java]: ci failed ',' character between ' '  white space.
You need to add a white space.

maybe
```
suggestions.add(new InterpreterCompletion(suggestedFinish,suggestedFinish));
```
to
```
suggestions.add(new InterpreterCompletion(suggestedFinish, suggestedFinish));
```

**Comment by bzz** [comment]
CI fails due to https://github.com/npm/npm/issues/13284#issuecomment-230799479

**Comment by cloverhearts** [comment]
@bzz 
in my opnnion..

this reason.
```
[ERROR] HDFSFileInterpreter.java[314:75] (whitespace) WhitespaceAfter: ',' is not followed by whitespace.
```
https://s3.amazonaws.com/archive.travis-ci.org/jobs/142795525/log.txt

**Comment by astroshim** [comment]
@cloverhearts Thank you very much!

**Comment by astroshim** [comment]
CI's passed. Thank you @bzz and @cloverhearts !

**Comment by felixcheung** [comment]
LGTM

**Comment by astroshim** [comment]
Can this PR going to be merged?

**Comment by bzz** [comment]
Merging if there is no more discussion

**Pull request closed without merging**